app/models/setting.rb

```diff
@@ -271,7 +279,7 @@ def self.cache
 
   def self.load_defaults
     # We may be executing something like rake db:migrate:reset, which destroys this table; only continue if the table exists
-    Setting.first rescue return false
+    Setting.table_exists? rescue return false
```

Comment:
`Setting.first` leads to firing a query every time we call load defaults, elsewhere we use the `table_exists?` technique which seems more sane